Allow relative path for JS loading

When I split my config into tasks folder, using a js config file, the following error is shown:
```
Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module 'tasks/postcss.js'
```
However, this does not seem to affect yaml configs.

After looking into this using `node --inspect`, the culprit seems to be [this](https://github.com/firstandthird/load-grunt-config/blob/e5f90e3fb838ee6356407fa2946eaac2e638a000/lib/readfile.js#L28):
```Javascript
    // JS / JSON / CoffeeScript
    if (ext.match(/json|js|coffee|ls/)) {
        return require(file);
    }
```
Obviously, "require" without "./" makes node mistake the file name as a module name.
The correct syntax should be ``require(`./${file}`)`` if the path is relative.

Fixing this should be easy, I will provide a patch soon.

--- a/lib/readfile.js
+++ b/lib/readfile.js
@@ -25,6 +25,9 @@ module.exports = function(file) {
 
     // JS / JSON / CoffeeScript
     if (ext.match(/json|js|coffee|ls/)) {
+        if (!path.isAbsolute(file)) {
+            file = path.join(process.cwd(), file)
+        }
         return require(file);
     }
 

